Move test utils from legacy api to agnostic

yuliia-fryshko opened the pull request:
## Summary

Closes [219684](https://github.com/elastic/kibana/issues/219684)

This PR removes the `observability_ai_assistant_api_integration` folder, relocating its tests and dependencies into relevant functional test directories or agnostic api tests. Common utilities used across both (functional and agnostic) suites have been moved to the agnostic folder.


### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.

- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)
- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [ ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [ ] If a plugin configuration key changed, check if it needs to be allowlisted in the cloud and added to the [docker list](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)
- [ ] This was checked for breaking HTTP API changes, and any breaking changes have been approved by the breaking-change committee. The `release_note:breaking` label should be applied in these situations.
- [ ] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed
- [ ] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

### Identify risks

Does this PR introduce any risks? For example, consider risks like hard to test bugs, performance regression, potential of data loss.

Describe the risk, its severity, and mitigation for each identified risk. Invite stakeholders and evaluate how to proceed before merging.

- [ ] [See some risk examples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)
- [ ] ...





<!--ONMERGE {"backportTargets":["9.1"]} ONMERGE-->

<!--ONMERGE {"backportTargets":["8.19","9.1"]} ONMERGE-->

yuliia-fryshko committed:
Make chat.spec.ts use only agnostic utils

yuliia-fryshko committed:
Removed observability_ai_assistant_api_integration folder, moved all dependencies to agnostic and functional tests folder

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/309801)
* Commit: e3e720aa22505546793214c7eecc4997d08ad650

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/309801#0197825b-d912-4c4c-bee9-e44ed67ad3db)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309801","number":309801,"commit":"e3e720aa22505546793214c7eecc4997d08ad650"}],"number":309801}
buildkite-pr-comment-->

jbudz reviewed (approved):
`.buildkite/ftr_oblt_stateful_configs.yml` LGTM

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310218)
* Commit: 084845d51604b2d7cd90a0007c3e550aa9729a44

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/310218#0197853f-19f1-4cfc-bcaf-673d04785ef8)

### History
* :broken_heart: [Build #310082](https://buildkite.com/elastic/kibana-pull-request/builds/310082) failed 5cf455e71048aa35b2abb07ec5dafab8495ed648
* :broken_heart: [Build #309911](https://buildkite.com/elastic/kibana-pull-request/builds/309911) failed 60c2a7e7d9df28d74b7ea74abdea53e3e60e10de
* :broken_heart: [Build #309801](https://buildkite.com/elastic/kibana-pull-request/builds/309801) failed e3e720aa22505546793214c7eecc4997d08ad650

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310218","number":310218,"commit":"084845d51604b2d7cd90a0007c3e550aa9729a44"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310082","number":310082,"commit":"5cf455e71048aa35b2abb07ec5dafab8495ed648"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309911","number":309911,"commit":"60c2a7e7d9df28d74b7ea74abdea53e3e60e10de"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309801","number":309801,"commit":"e3e720aa22505546793214c7eecc4997d08ad650"}],"number":310218}
buildkite-pr-comment-->

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint --no-cache --fix'

viduni94 commented on x-pack/test/observability_ai_assistant_functional/common/config.ts:
```diff
@@ -5,29 +5,117 @@
+      getScopedApiClientForUsername: () => getScopedApiClientForUsername,
+      apmSynthtraceEsClient: (context: InheritedFtrProviderContext) =>
+        getApmSynthtraceEsClient(context, apmSynthtraceKibanaClient),
+      observabilityAIAssistantAPIClient: async () => {
```
Is this being used anywhere? If not, can we remove it?

viduni94 commented on x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/utils/observability_ai_assistant_api_client.ts:
Is this file needed? Are we using the API client in this anywhere?
The deployment agnostic API client is in `x-pack/test/api_integration/deployment_agnostic/services/observability_ai_assistant_api.ts`

viduni94 commented on .github/CODEOWNERS:
```diff
@@ -1338,7 +1338,6 @@ src/platform/plugins/shared/discover/public/context_awareness/profile_providers/
 /x-pack/test_serverless/functional/page_objects/svl_data_usage.ts @elastic/kibana-management
-/x-pack/test/observability_ai_assistant_api_integration @elastic/obs-ai-assistant
 /x-pack/test/observability_ai_assistant_functional @elastic/obs-ai-assistant
 /x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai-assistant
```
Can we remove everything in `test_serverless` too since we moved to deployment_agnostic?

<img width="597" alt="image" src="https://github.com/user-attachments/assets/0b528b8f-b485-4734-8744-a22d312d6a39" />

yuliia-fryshko committed:
removed unused test suites

- The deployment agnostic API client is in x-pack/test/api_integration/deployment_agnostic/services/observability_ai_assistant_api.ts;
-test_serverless/api_integration/test_suites/observability/ai_assistant (since we moved to deployment_agnostic)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311299)
* Commit: 6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/311299#01979bff-fb91-4811-88af-e7ae26213877)

### History
* :green_heart: [Build #310285](https://buildkite.com/elastic/kibana-pull-request/builds/310285) succeeded 7ce5728f17fae6e58958fb4ae9e011de412e0c3b
* :broken_heart: [Build #310218](https://buildkite.com/elastic/kibana-pull-request/builds/310218) failed 084845d51604b2d7cd90a0007c3e550aa9729a44
* :broken_heart: [Build #310082](https://buildkite.com/elastic/kibana-pull-request/builds/310082) failed 5cf455e71048aa35b2abb07ec5dafab8495ed648
* :broken_heart: [Build #309911](https://buildkite.com/elastic/kibana-pull-request/builds/309911) failed 60c2a7e7d9df28d74b7ea74abdea53e3e60e10de
* :broken_heart: [Build #309801](https://buildkite.com/elastic/kibana-pull-request/builds/309801) failed e3e720aa22505546793214c7eecc4997d08ad650

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311299","number":311299,"commit":"6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310285","number":310285,"commit":"7ce5728f17fae6e58958fb4ae9e011de412e0c3b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310218","number":310218,"commit":"084845d51604b2d7cd90a0007c3e550aa9729a44"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310082","number":310082,"commit":"5cf455e71048aa35b2abb07ec5dafab8495ed648"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309911","number":309911,"commit":"60c2a7e7d9df28d74b7ea74abdea53e3e60e10de"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309801","number":309801,"commit":"e3e720aa22505546793214c7eecc4997d08ad650"}],"number":311299}
buildkite-pr-comment-->

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/lint_ts_projects --fix'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311340)
* Commit: 0c6c6d81e7cc210f65837a6f7ec75e95f81628d7
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-224381-0c6c6d81e7cc`

### History
* :broken_heart: [Build #311299](https://buildkite.com/elastic/kibana-pull-request/builds/311299) failed 6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85
* :green_heart: [Build #310285](https://buildkite.com/elastic/kibana-pull-request/builds/310285) succeeded 7ce5728f17fae6e58958fb4ae9e011de412e0c3b
* :broken_heart: [Build #310218](https://buildkite.com/elastic/kibana-pull-request/builds/310218) failed 084845d51604b2d7cd90a0007c3e550aa9729a44
* :broken_heart: [Build #310082](https://buildkite.com/elastic/kibana-pull-request/builds/310082) failed 5cf455e71048aa35b2abb07ec5dafab8495ed648
* :broken_heart: [Build #309911](https://buildkite.com/elastic/kibana-pull-request/builds/309911) failed 60c2a7e7d9df28d74b7ea74abdea53e3e60e10de

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311340","number":311340,"commit":"0c6c6d81e7cc210f65837a6f7ec75e95f81628d7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311299","number":311299,"commit":"6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310285","number":310285,"commit":"7ce5728f17fae6e58958fb4ae9e011de412e0c3b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310218","number":310218,"commit":"084845d51604b2d7cd90a0007c3e550aa9729a44"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310082","number":310082,"commit":"5cf455e71048aa35b2abb07ec5dafab8495ed648"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309911","number":309911,"commit":"60c2a7e7d9df28d74b7ea74abdea53e3e60e10de"}],"number":311340}
buildkite-pr-comment-->

csr commented on x-pack/test/observability_ai_assistant_functional/common/config.ts:
```diff
@@ -5,29 +5,108 @@
+        getApmSynthtraceEsClient(context, apmSynthtraceKibanaClient),
+    },
+    junit: {
+      reportName: `Observability AI Assistant API Integration tests (${name})`,
```
It looks like we're creating an API integration test config here in the _functional_ test plugin. AFAIK, unlike UI tests, API integration tests aren't considered functional tests. Is this intended?

viduni94 reviewed (commented):
Thanks for removing the `test_serverless/**/test_suites/observability/ai_assistant` test folder @yuliia-fryshko 

Can you please remove all references to it as well?
I see it being referred in
- CODEOWNERS file
- `.buildkite/ftr_oblt_serverless_configs.yml` file

viduni94 commented on x-pack/test/observability_ai_assistant_functional/common/config.ts:
```diff
@@ -5,29 +5,108 @@
 export type CreateTestConfig = ReturnType<typeof createTestConfig>;
 export type TestConfig = Awaited<ReturnType<typeof getTestConfig>>;
 
+export function createObservabilityAIAssistantAPIConfig({
```
Why do we have a `createObservabilityAIAssistantAPIConfig` and `getTestConfig`?
AFAICT, only `apmSynthtraceEsClient` from `createObservabilityAIAssistantAPIConfig` is being used in functional tests (correct me if I'm wrong). If that's the case, can we move `apmSynthtraceEsClient` to `getTestConfig` and move everything else that's needed from the base config to `getTestConfig` and completely get rid of `createObservabilityAIAssistantAPIConfig`? Would that work?

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/312745)
* Commit: 745892fcff5cb745f992989c13d4dceb1be39845

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/312745#0197a7d3-4348-4c44-8ed7-6c6e5bb3dffe)

### History
* :broken_heart: [Build #311340](https://buildkite.com/elastic/kibana-pull-request/builds/311340) failed 0c6c6d81e7cc210f65837a6f7ec75e95f81628d7
* :broken_heart: [Build #311299](https://buildkite.com/elastic/kibana-pull-request/builds/311299) failed 6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85
* :green_heart: [Build #310285](https://buildkite.com/elastic/kibana-pull-request/builds/310285) succeeded 7ce5728f17fae6e58958fb4ae9e011de412e0c3b
* :broken_heart: [Build #310218](https://buildkite.com/elastic/kibana-pull-request/builds/310218) failed 084845d51604b2d7cd90a0007c3e550aa9729a44
* :broken_heart: [Build #310082](https://buildkite.com/elastic/kibana-pull-request/builds/310082) failed 5cf455e71048aa35b2abb07ec5dafab8495ed648

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312745","number":312745,"commit":"745892fcff5cb745f992989c13d4dceb1be39845"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311340","number":311340,"commit":"0c6c6d81e7cc210f65837a6f7ec75e95f81628d7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311299","number":311299,"commit":"6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310285","number":310285,"commit":"7ce5728f17fae6e58958fb4ae9e011de412e0c3b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310218","number":310218,"commit":"084845d51604b2d7cd90a0007c3e550aa9729a44"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310082","number":310082,"commit":"5cf455e71048aa35b2abb07ec5dafab8495ed648"}],"number":312745}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/312759)
* Commit: 667ab651e03e9fc18224a5909d7761ee709ba52e

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/312759#0197a7ea-64aa-4ebe-819f-b28dfd18204c)

### History
* :broken_heart: [Build #312745](https://buildkite.com/elastic/kibana-pull-request/builds/312745) failed 745892fcff5cb745f992989c13d4dceb1be39845
* :broken_heart: [Build #311340](https://buildkite.com/elastic/kibana-pull-request/builds/311340) failed 0c6c6d81e7cc210f65837a6f7ec75e95f81628d7
* :broken_heart: [Build #311299](https://buildkite.com/elastic/kibana-pull-request/builds/311299) failed 6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85
* :green_heart: [Build #310285](https://buildkite.com/elastic/kibana-pull-request/builds/310285) succeeded 7ce5728f17fae6e58958fb4ae9e011de412e0c3b
* :broken_heart: [Build #310218](https://buildkite.com/elastic/kibana-pull-request/builds/310218) failed 084845d51604b2d7cd90a0007c3e550aa9729a44
* :broken_heart: [Build #310082](https://buildkite.com/elastic/kibana-pull-request/builds/310082) failed 5cf455e71048aa35b2abb07ec5dafab8495ed648

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312759","number":312759,"commit":"667ab651e03e9fc18224a5909d7761ee709ba52e"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312745","number":312745,"commit":"745892fcff5cb745f992989c13d4dceb1be39845"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311340","number":311340,"commit":"0c6c6d81e7cc210f65837a6f7ec75e95f81628d7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311299","number":311299,"commit":"6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310285","number":310285,"commit":"7ce5728f17fae6e58958fb4ae9e011de412e0c3b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310218","number":310218,"commit":"084845d51604b2d7cd90a0007c3e550aa9729a44"}],"number":312759}
buildkite-pr-comment-->

yuliia-fryshko committed:
removed unneeded createObservabilityAIAssistantAPIConfig

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/312870)
* Commit: fde1e4a6cbc24d5ed6a748816a9a9234b8c26a2e

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/312870#0197a93a-a604-4c28-99ca-aa16930daf2b)

### History
* :broken_heart: [Build #312759](https://buildkite.com/elastic/kibana-pull-request/builds/312759) failed 667ab651e03e9fc18224a5909d7761ee709ba52e
* :broken_heart: [Build #312745](https://buildkite.com/elastic/kibana-pull-request/builds/312745) failed 745892fcff5cb745f992989c13d4dceb1be39845
* :broken_heart: [Build #311340](https://buildkite.com/elastic/kibana-pull-request/builds/311340) failed 0c6c6d81e7cc210f65837a6f7ec75e95f81628d7
* :broken_heart: [Build #311299](https://buildkite.com/elastic/kibana-pull-request/builds/311299) failed 6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85
* :green_heart: [Build #310285](https://buildkite.com/elastic/kibana-pull-request/builds/310285) succeeded 7ce5728f17fae6e58958fb4ae9e011de412e0c3b

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312870","number":312870,"commit":"fde1e4a6cbc24d5ed6a748816a9a9234b8c26a2e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312759","number":312759,"commit":"667ab651e03e9fc18224a5909d7761ee709ba52e"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312745","number":312745,"commit":"745892fcff5cb745f992989c13d4dceb1be39845"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311340","number":311340,"commit":"0c6c6d81e7cc210f65837a6f7ec75e95f81628d7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311299","number":311299,"commit":"6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310285","number":310285,"commit":"7ce5728f17fae6e58958fb4ae9e011de412e0c3b"}],"number":312870}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/312873)
* Commit: a4aa78345fb52d2c2a8b682a7078a4f23b7e830d

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/312873#0197a950-d026-4129-b1eb-0602e6ddc7b8)

### History
* :broken_heart: [Build #312870](https://buildkite.com/elastic/kibana-pull-request/builds/312870) failed fde1e4a6cbc24d5ed6a748816a9a9234b8c26a2e
* :broken_heart: [Build #312759](https://buildkite.com/elastic/kibana-pull-request/builds/312759) failed 667ab651e03e9fc18224a5909d7761ee709ba52e
* :broken_heart: [Build #312745](https://buildkite.com/elastic/kibana-pull-request/builds/312745) failed 745892fcff5cb745f992989c13d4dceb1be39845
* :broken_heart: [Build #311340](https://buildkite.com/elastic/kibana-pull-request/builds/311340) failed 0c6c6d81e7cc210f65837a6f7ec75e95f81628d7
* :broken_heart: [Build #311299](https://buildkite.com/elastic/kibana-pull-request/builds/311299) failed 6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85
* :green_heart: [Build #310285](https://buildkite.com/elastic/kibana-pull-request/builds/310285) succeeded 7ce5728f17fae6e58958fb4ae9e011de412e0c3b

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312873","number":312873,"commit":"a4aa78345fb52d2c2a8b682a7078a4f23b7e830d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312870","number":312870,"commit":"fde1e4a6cbc24d5ed6a748816a9a9234b8c26a2e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312759","number":312759,"commit":"667ab651e03e9fc18224a5909d7761ee709ba52e"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312745","number":312745,"commit":"745892fcff5cb745f992989c13d4dceb1be39845"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311340","number":311340,"commit":"0c6c6d81e7cc210f65837a6f7ec75e95f81628d7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311299","number":311299,"commit":"6cd8fbdc89f2503c144d129a7f1ef02dbb7a8d85"}],"number":312873}
buildkite-pr-comment-->

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint --no-cache --fix'

yuliia-fryshko committed:
Merge branch 'main' into move-test-utils-from-legacy-api-to-agnostic

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/312977)
* Commit: a65cd9f0f538743506b912524a8d69d345bdb42a
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-224381-a65cd9f0f538`

### History
* :yellow_heart: [Build #312919](https://buildkite.com/elastic/kibana-pull-request/builds/312919) was flaky 7201279cabbfce76a40d9e8f79b7e8d7380a63ee
* :broken_heart: [Build #312873](https://buildkite.com/elastic/kibana-pull-request/builds/312873) failed a4aa78345fb52d2c2a8b682a7078a4f23b7e830d
* :broken_heart: [Build #312870](https://buildkite.com/elastic/kibana-pull-request/builds/312870) failed fde1e4a6cbc24d5ed6a748816a9a9234b8c26a2e
* :broken_heart: [Build #312759](https://buildkite.com/elastic/kibana-pull-request/builds/312759) failed 667ab651e03e9fc18224a5909d7761ee709ba52e
* :broken_heart: [Build #312745](https://buildkite.com/elastic/kibana-pull-request/builds/312745) failed 745892fcff5cb745f992989c13d4dceb1be39845

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312977","number":312977,"commit":"a65cd9f0f538743506b912524a8d69d345bdb42a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312919","number":312919,"commit":"7201279cabbfce76a40d9e8f79b7e8d7380a63ee"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312873","number":312873,"commit":"a4aa78345fb52d2c2a8b682a7078a4f23b7e830d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312870","number":312870,"commit":"fde1e4a6cbc24d5ed6a748816a9a9234b8c26a2e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312759","number":312759,"commit":"667ab651e03e9fc18224a5909d7761ee709ba52e"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312745","number":312745,"commit":"745892fcff5cb745f992989c13d4dceb1be39845"}],"number":312977}
buildkite-pr-comment-->

yuliia-fryshko committed:
Merge branch 'main' into move-test-utils-from-legacy-api-to-agnostic

viduni94 reviewed (approved):
Added a couple comments.
Overall LGTM. Thanks for cleaning up the test utils @yuliia-fryshko

yuliia-fryshko committed:
Removed commented code

yuliia-fryshko committed:
Renamed the report of functional tests

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/313301)
* Commit: b6af243a346b58fe62559d2eaa6f2bd3c9b56ad5
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-224381-b6af243a346b`

### History
* :green_heart: [Build #313025](https://buildkite.com/elastic/kibana-pull-request/builds/313025) succeeded 3a4b88acc1718f58095e6cf8c4836c0e72a27cdd
* :green_heart: [Build #312977](https://buildkite.com/elastic/kibana-pull-request/builds/312977) succeeded a65cd9f0f538743506b912524a8d69d345bdb42a
* :yellow_heart: [Build #312919](https://buildkite.com/elastic/kibana-pull-request/builds/312919) was flaky 7201279cabbfce76a40d9e8f79b7e8d7380a63ee
* :broken_heart: [Build #312873](https://buildkite.com/elastic/kibana-pull-request/builds/312873) failed a4aa78345fb52d2c2a8b682a7078a4f23b7e830d
* :broken_heart: [Build #312870](https://buildkite.com/elastic/kibana-pull-request/builds/312870) failed fde1e4a6cbc24d5ed6a748816a9a9234b8c26a2e

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/313301","number":313301,"commit":"b6af243a346b58fe62559d2eaa6f2bd3c9b56ad5"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/313025","number":313025,"commit":"3a4b88acc1718f58095e6cf8c4836c0e72a27cdd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312977","number":312977,"commit":"a65cd9f0f538743506b912524a8d69d345bdb42a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312919","number":312919,"commit":"7201279cabbfce76a40d9e8f79b7e8d7380a63ee"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312873","number":312873,"commit":"a4aa78345fb52d2c2a8b682a7078a4f23b7e830d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312870","number":312870,"commit":"fde1e4a6cbc24d5ed6a748816a9a9234b8c26a2e"}],"number":313301}
buildkite-pr-comment-->

yuliia-fryshko committed:
Merge branch 'main' into move-test-utils-from-legacy-api-to-agnostic

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/313351)
* Commit: 2a1ec965c450326ff5f8b37ffe150b762b064ec7
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-224381-2a1ec965c450`

### History
* :yellow_heart: [Build #313301](https://buildkite.com/elastic/kibana-pull-request/builds/313301) was flaky b6af243a346b58fe62559d2eaa6f2bd3c9b56ad5
* :green_heart: [Build #313025](https://buildkite.com/elastic/kibana-pull-request/builds/313025) succeeded 3a4b88acc1718f58095e6cf8c4836c0e72a27cdd
* :green_heart: [Build #312977](https://buildkite.com/elastic/kibana-pull-request/builds/312977) succeeded a65cd9f0f538743506b912524a8d69d345bdb42a
* :yellow_heart: [Build #312919](https://buildkite.com/elastic/kibana-pull-request/builds/312919) was flaky 7201279cabbfce76a40d9e8f79b7e8d7380a63ee
* :broken_heart: [Build #312873](https://buildkite.com/elastic/kibana-pull-request/builds/312873) failed a4aa78345fb52d2c2a8b682a7078a4f23b7e830d

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/313351","number":313351,"commit":"2a1ec965c450326ff5f8b37ffe150b762b064ec7"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/313301","number":313301,"commit":"b6af243a346b58fe62559d2eaa6f2bd3c9b56ad5"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/313025","number":313025,"commit":"3a4b88acc1718f58095e6cf8c4836c0e72a27cdd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312977","number":312977,"commit":"a65cd9f0f538743506b912524a8d69d345bdb42a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312919","number":312919,"commit":"7201279cabbfce76a40d9e8f79b7e8d7380a63ee"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312873","number":312873,"commit":"a4aa78345fb52d2c2a8b682a7078a4f23b7e830d"}],"number":313351}
buildkite-pr-comment-->

sorenlouv reviewed (approved):
thank you!

yuliia-fryshko committed:
Merge branch 'main' into move-test-utils-from-legacy-api-to-agnostic

yuliia-fryshko committed:
Merge branch 'main' into move-test-utils-from-legacy-api-to-agnostic

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/313645)
* Commit: f91c3c75a449c2c3793c1bd8521381751266e1c5
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-224381-f91c3c75a449`

### History
* :green_heart: [Build #313351](https://buildkite.com/elastic/kibana-pull-request/builds/313351) succeeded 2a1ec965c450326ff5f8b37ffe150b762b064ec7
* :yellow_heart: [Build #313301](https://buildkite.com/elastic/kibana-pull-request/builds/313301) was flaky b6af243a346b58fe62559d2eaa6f2bd3c9b56ad5
* :green_heart: [Build #313025](https://buildkite.com/elastic/kibana-pull-request/builds/313025) succeeded 3a4b88acc1718f58095e6cf8c4836c0e72a27cdd
* :green_heart: [Build #312977](https://buildkite.com/elastic/kibana-pull-request/builds/312977) succeeded a65cd9f0f538743506b912524a8d69d345bdb42a
* :yellow_heart: [Build #312919](https://buildkite.com/elastic/kibana-pull-request/builds/312919) was flaky 7201279cabbfce76a40d9e8f79b7e8d7380a63ee
* :broken_heart: [Build #312873](https://buildkite.com/elastic/kibana-pull-request/builds/312873) failed a4aa78345fb52d2c2a8b682a7078a4f23b7e830d

cc @yuliia-fryshko

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/313645","number":313645,"commit":"f91c3c75a449c2c3793c1bd8521381751266e1c5"},{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/313351","number":313351,"commit":"2a1ec965c450326ff5f8b37ffe150b762b064ec7"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/313301","number":313301,"commit":"b6af243a346b58fe62559d2eaa6f2bd3c9b56ad5"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/313025","number":313025,"commit":"3a4b88acc1718f58095e6cf8c4836c0e72a27cdd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312977","number":312977,"commit":"a65cd9f0f538743506b912524a8d69d345bdb42a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312919","number":312919,"commit":"7201279cabbfce76a40d9e8f79b7e8d7380a63ee"}],"number":313645}
buildkite-pr-comment-->

kibanamachine commented:
Friendly reminder: Looks like this PR hasn’t been backported yet.
To create automatically backports add a `backport:*` label or prevent reminders by adding the `backport:skip` label.
You can also create backports manually by running `node scripts/backport --pr 224381` locally
cc: @yuliia-fryshko

kibanamachine commented:
Friendly reminder: Looks like this PR hasn’t been backported yet.
To create automatically backports add a `backport:*` label or prevent reminders by adding the `backport:skip` label.
You can also create backports manually by running `node scripts/backport --pr 224381` locally
cc: @yuliia-fryshko

kibanamachine commented:
Starting backport for target branches: 9.1

https://github.com/elastic/kibana/actions/runs/16081123349

kibanamachine commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|9.1|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226667">](https://github.com/elastic/kibana/pull/226667)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

kibanamachine commented:
Starting backport for target branches: 8.19, 9.1

https://github.com/elastic/kibana/actions/runs/16081790737

kibanamachine commented:
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|8.19|Backport failed because of merge conflicts|
|✅|9.1|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/226667">](https://github.com/elastic/kibana/pull/226667)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Manual backport
To create the backport manually run:
```
node scripts/backport --pr 224381
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->
